### DISCUSSION TITLE
Suscpicion of network scan action

### DISCUSSION THREAD

**Discussion by akoita** [apache/couchdb#3627] [General]
Hi,

We use CouchDB version 2.3.1 through Hyperledger Fabric 1.4.9. We are using it as a Docker container.

Our security team informs us that they have detected network scanning actions from some processes in our deployment, and these processes appear to match CouchDB (port number).

Have there been any such reports before? Can you confirm that CouchDB does perform network scan type actions in its operation? And if so, isn't that dangerous from a security perspective?

Thanks for your help.

**Comment by wohali** [comment]
CouchDB does not perform network scan actions during operation. The only outgoing connections from CouchDB are during user-requested replications to other CouchDB instances. Perhaps these are being detected as "network scans?"

Can your security team provide any additional detail?

**Reply by akoita** [reply]
Thank you for this information. Okay, I will try to get more details about the suspected behavior with our security team.
